tests/test.sh

```diff
@@ -144,5 +147,19 @@ do
   echo -e "\n\n=====Execution Node $i Log====="
   cat /tmp/log/execute_node"$i".log
 done
-# TODO: stop the autolab services and remove the logs
+
+# Stop all services started by AutolabJS
```

Comment:
This is a nice idea. You might want to load up the port configuration information from `deploy/configs/` using json shell utility.
Another idea to aid the readability is to use associative array to create a map of <component,port> and then invoke the function on all the active components.